ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -500,11 +505,15 @@ def _parse_data_model_upstream(self, mnode):
             for node in mnode["depends_on"]["nodes"]:
                 try:
                     _, database, table = node.split(".", 2)
-                    table_fqn = get_fqdn(
-                        Table,
+                    # TODO: FIXME
+                    table_fqn = fqn.build(
+                        self.metadata,
+                        entity_type=Table,
                         service_name=self.config.serviceName,
+                        database_name=database,
+                        schema_name=None,
```

Comment:
@pmbrull below is an example of the `nodes` list that we get from the manifest.json file in DBT. 
`"nodes": ["model.jaffle_shop.stg_orders", "model.jaffle_shop.stg_payments"]`

we do not get the schema name here so this is looks fine to me.